index.html

```diff
@@ -5,40 +5,263 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css"/>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
```

Comment:
- [ ] [Optional] Great job on your work! However, there is a small issue with the vertical scroll bar appearing on your webpage. According to the design, there should be no vertical scroll bar visible. Please review and adjust the width and spacing of your elements to ensure that the content fits within the available space without triggering the appearance of a vertical scroll bar. By resolving this issue, you will achieve a closer match to the intended design and improve the overall user experience. 👍 

[Document (9).webm](https://github.com/MayPyone/My_Portfolio/assets/55840999/749828a6-39ac-4b2d-bbde-5a3b6b8a3c8e)